src/main/java/com/serjltt/moshi/adapters/Wrapped.java

```diff
@@ -64,22 +64,37 @@
 @Documented
 @JsonQualifier
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER })
+@Target({ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER})
 public @interface Wrapped {
   /** The path to the wrapped json value. */
-  String[] value();
+  String[] path();
```

Comment:
Or make it two different annotations. Wrapped stays the way it is right now. And then there's another one that fails. Under the hood they both can use the same adapter logic.